actors/builtin/paych/paych_actor.go

```diff
@@ -172,6 +172,10 @@ func (pca Actor) UpdateChannelState(rt vmr.Runtime, params *UpdateChannelStatePa
 		rt.Abortf(exitcode.ErrIllegalArgument, "this voucher has expired!")
 	}
 
+	if sv.Amount.Sign() <= 0 {
```

Comment:
TODO: Do we want to assert somewhere that amount > the previous amount?

Reply:
I would think yes. If not, I think we'd need something where balanceDelta is computed in any case. @whyrusleeping could you confirm?

If correct behaviour with a smaller-than-previous amount is not already tested, could you please add tests to validate/demonstrate behaviour?

Reply:
Nope, The amount can move in any direction, it just can't be negative.